hazelcast/src/main/java/com/hazelcast/nio/NodeIOService.java

```diff
@@ -332,5 +382,39 @@ private void transformPortDefinitionsToPorts(Collection<String> portDefinitions,
         return networkConfig.getOutboundPortDefinitions() == null
                 ? Collections.<String>emptySet() : networkConfig.getOutboundPortDefinitions();
     }
+
+    private class ThroughputMonitor extends Thread {
+        @Override
+        public void run() {
+            while (!interrupted()) {
+                int operationPackets = operationPacketsReceived.getAndSet(0);
+                int eventPackets = eventPacketsReceived.getAndSet(0);
+                int wantPackets = wanPacketsReceived.getAndSet(0);
+                int clientPackets = clientPacketsReceived.getAndSet(0);
+
+                ILogger logger = node.loggingService.getLogger(ThroughputMonitor.class);
+                if (logger != null) {
+                    StringBuilder sb = new StringBuilder(Tracing.PACKET_THROUGHPUT_CODE)
```

Comment:
In the future we'll probably want to have a dedicates file where everything related to health/performance etc is stored.